src/useDescriptors.tsx

```diff
@@ -52,6 +66,7 @@ export default function useDescriptors({
                 screen={screen}
                 getState={getState}
                 setState={setState}
+                getKey={() => route.key}
```

Comment:
If we pass it inline here, it'll invalidate the memo for `navigation` prop in `SceneView` every render.

Reply:
Though if all these function does is `() => route.key`, shouldn't using `route.key` directly work? 🤔